nb2workflow/nbadapter.py

```diff
@@ -813,9 +813,16 @@ def download_file(self, file_url, tmpdir):
     def handle_url_params(self, parameters, tmpdir, context={}):
         adapted_parameters = copy.deepcopy(parameters)
         exceptions = []
+        ontology_available = False
+        if len(ontology.g) > 0:
+            ontology_available = True
         for input_par_name, input_par_obj in self.input_parameters.items():
-            # TODO use oda_api.ontology_helper
-            if input_par_obj['owl_type'] == f"{oda_prefix}POSIXPath":
+            if ontology_available:
+                parameter_hierarchy = ontology.get_parameter_hierarchy(input_par_obj['owl_type'])
+                is_posix_path = f"{oda_prefix}POSIXPath" in parameter_hierarchy
+            else:
+                is_posix_path = f"{oda_prefix}POSIXPath" in input_par_obj['owl_type']
```

Comment:
This only works in very basic situation, when the annotation is strictly `# oda:POSIXPath`
If one adds anything on top, e.g. `oda:label`, it will break. Could you add such variants to the test notebook?

Reply:
test with extra annotations added, and I see the problem

Reply:
I added a test with extra annotations (and ontology available), and it works

Reply:
btw, which check do you think it is more appropriate in case the ontology was not available ?

Reply:
I think this can be implemented in ontology module. 
I thought about fallback behaviour of `get_parameter_hierarchy` in case there is no "main" ontology loaded, to assume that argument is parameter (because it can't be verified) and just returning a hierarchy of all its parent classes. Then the if clause here won't be needed.